cmd/test/cmd.go

```diff
@@ -308,12 +308,11 @@ func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtr
 		if err := manifest.ExpandEnvVars(); err != nil {
 			return analytics.TestMetadata{}, fmt.Errorf("failed to expand manifest environment variables: %w", err)
 		}
-		oktetoLog.Information("'%s' was already deployed. To redeploy run 'okteto deploy'", devenvName)
```

Comment:
I've removed this log now because when not passing the `--deploy` flag, we are not deploying, but only expanding vars. I find confusing this log. Maybe we can add another one for when a user uses `--deploy` on an application that was already deployed and we inform that it was already deployed. Not sure tho. Thoughts @pchico83 @ifbyol ?

Reply:
I think it's fine to remove it